src/main/java/hudson/remoting/ClassFilter.java

```diff
@@ -228,14 +226,6 @@ public static void appendDefaultFilter(Pattern filter) throws ClassFilterExcepti
             return patterns;
         } catch (IOException ex) {
             throw new Error("Could not load user provided overrides for ClassFiltering from as "+prop+" does not exist or is not readable.",ex);
-        } finally {
-            if (br != null) {
-                try {
-                    br.close();
-                } catch (IOException ioEx) {
-                    LOGGER.log(Level.WARNING, "Failed to cleanly close input stream", ioEx);
```

Comment:
It changes the code behavior. Instead of a log warning, close operation failure will result in an `Error` being thrown